docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Namespace extension
+weight: 3
+---
+### Introduction
+
+The Liqo namespace model extends the K8s namespace concepts introducing the support to remote namespaces, "twins" of the local namespace. 
+Those "twin" namespaces map the home cluster namespace on a remote cluster and its associated resources (e.g., services, configmaps). 
+Regarding pod offloading, remote namespace hosts the offloaded pods belonging to the home twin namespace, as they were executed in the home cluster.
+
+### Quick Offloading
+
+Since the very first version, Liqo provides a label-based mechanism to quickly set-up namespace offloading over remote clusters.
+
+To enable it, you should label a target namespace:
+
+```bash
+kubectl label namespace demo-namespace liqo.io/enabled=true
+```
+
+Pods scheduled in the "*demo-namespace*" will be potentially offloaded inside remote clusters. 
+With *quick offloading*, we are selecting all peered clusters suitable for offloading. 
+If you need a fine-grained approach, you can rely on custom offloading and the *namespaceOffloading* resource.
+
+### Custom Offloading
+
+As mentioned before, Liqo namespace enables the extension of the standard *v1.Namespace* over remote clusters. 
+To control all different aspects of the namespace extension, Liqo provides a *namespaceOffloading* resource. 
+The policies defined inside the *namespaceOffloading* specify how the local namespace can be replicated on peered clusters.
+
+Any namespace inside the home cluster can have its own *namespaceOffloading* resource and its corresponding deployment policy.
```

Comment:
```suggestion
Any namespace inside the home cluster can have its own *namespaceOffloading* resource and its corresponding offloading policy.
```

Would join (at least) the next sentence to this paragraph, to avoid single sentence paragraphs.